capture/src/main/kotlin/com/alfresco/capture/CameraFragment.kt

```diff
@@ -1,15 +1,15 @@
 package com.alfresco.capture
 
+import android.Manifest
 import android.annotation.SuppressLint
+import android.content.pm.PackageManager
 import android.graphics.Color
 import android.graphics.drawable.ColorDrawable
+import android.location.Location
 import android.net.Uri
 import android.os.Bundle
-import android.view.KeyEvent
-import android.view.LayoutInflater
-import android.view.MotionEvent
-import android.view.View
-import android.view.ViewGroup
+import android.os.Looper
+import android.view.*
```

Comment:
Prefer using single import instead of wildcard.

Reply:
Done

Reply:
This is still wildcard. How is it changed?

Reply:
Done.